tests/integrations/tests/integrations/test_anthropic.py

```
@@ -537,54 +558,269 @@ def test_11_integration_specific_features(self, anthropic_client, test_config):
     @skip_if_no_api_key("anthropic")
     def test_12_error_handling_invalid_roles(self, anthropic_client, test_config):
```

Ruff (0.14.2)

Unused method argument: `test_config`

(ARG002)